applying_to_uol/README.md

```diff
@@ -0,0 +1,95 @@
+
+## Recognition of Prior Learning (RPL) Application
+
+RPL is the exemption of modules taken if you had taken it before in a prior diploma or university. It can be seen as a form of credit transfer. Full regulations can be found in the [course regulations](https://london.ac.uk/sites/default/files/regulations/progregs-computer-science-2019-2020.pdf). Grades attained from the programme will not be transferred, however a listing of the grade will be written on your transcript.
+
+You will have stated your intention to apply for RPL in your application form, and upon acceptance you will have been given detailed instructions on how to go about submitting proof for RPL.
+
+There is cost and limitations involved, currently it costs 392 GBP to get all 8 modules from the first year exempted. Likewise, you are only able to get first year modules exempted.
+
+You can get RPL credit in 3 of the following ways:
+
+1. [Automatic RPL: You hold a diploma from these colleges/are transferring from one of UoL's on-campus programmes](https://london.ac.uk/applications/how-apply/recognition-prior-learning/recognition-and-accreditation-prior-learning-3)
+2. [Discretionary RPL: You have a diploma/incomplete degree you would like the programme director to consider for RPL](https://london.ac.uk/applications/how-apply/recognition-prior-learning/recognition-and-accreditation-prior-learning-3)
+3. [Google's IT Diploma: You completed the course and would like the HCW exemption](https://www.coursera.org/professional-certificates/google-it-support)
+
+RPL from a discretionary route will take awhile, students have reported the process taking a full month before the process was complete. It would be wise to apply as soon as possible to avoid delays with module registration.
+
+## FAQs
+
+**Q:** My application to UoL/for RPL/some other administrative process is taking so long, have they received it/why's it taking so long?
+
+**A:** It takes forever. Sorry mate. That's why it's important to apply early for peace of mind. It means that your application isn't clogged up and done when professors aren't stressed out when they read your application.  
+
+RPL has taken a month, UoL admissions has taken longer. Write a nice email to nudge them whenever it's taken too long - ensuring that they have all you need done when they get around to your application.
+
+**Q:** What does my degree look like?
+
+**A:** It looks like [this](https://www.thestudentroom.co.uk/showthread.php?t=2459201).
+
+**Q:** Where would we be graduating?
+
+**A:** [Here, at The Barbican in London](https://www.youtube.com/watch?v=Oja7n2Kq2do)
+
+## Additional Resources
+
+[UoL Unofficial Students' Discord](https://discord.gg/fugMPVR)
+
+[UoL Facebook](https://www.facebook.com/LondonU/?ref=br_rs)
```

Comment:
Preferably prefix with hyphen for consistency:
- [UoL Facebook](https://www.facebook.com/LondonU/?ref=br_rs)